src/ucp/core/ucp_mm.c

```diff
@@ -106,6 +106,26 @@ static ucs_status_t ucp_memh_reg_mds(ucp_context_h context, ucp_mem_h memh,
     return UCS_OK;
 }
 
+ucs_status_t ucp_memory_type_detect_mds(ucp_context_h context, void *addr, size_t length,
+                                        uct_memory_type_t *mem_type)
```

Comment:
would suggest to rename mem_type to mem_type_p

Reply:
fixed